.github/workflows/test-environment.yml

```diff
@@ -483,6 +498,7 @@ jobs:
           aws s3 cp "./kspm_d4c.yaml" "${{ env.S3_BUCKET }}/kspm_d4c.yaml"
           aws s3 cp "./kspm_eks.yaml" "${{ env.S3_BUCKET }}/kspm_eks.yaml"
           aws s3 cp "./cspm-linux.sh" "${{ env.S3_BUCKET }}/cspm-linux.sh"
+          aws s3 cp "./aws-asset-inventory-linux.sh" "${{ env.S3_BUCKET }}/aws-asset-inventory-linux.sh"
```

Comment:
```suggestion
          aws s3 cp "./aws-asset-inventory-linux.sh" "$S3_BUCKET/aws-asset-inventory-linux.sh"
```

Reply:
Should I fix it for the entire file, since it's a direct copy of lines above?

Reply:
I'm not certain about the entire file, but at least in this action, yes, please apply the fix. In general, ${{ env. }} should be replaced wherever we are using inline scripting in the actions.